lib/danger/request_sources/github.rb

```diff
@@ -50,9 +50,16 @@ def update_pull_request!(warnings: nil, errors: nil, messages: nil)
         # Just remove the comment, if there's nothing to say.
         delete_old_comments!
       else
+        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id)
+        editable_issues = issues.reject { |issue|  issue[:body].include?("generated_by_danger") == false }
         body = generate_comment(warnings: warnings, errors: errors, messages: messages)
-        comment_result = client.add_comment(ci_source.repo_slug, ci_source.pull_request_id, body)
-        delete_old_comments!(except: comment_result[:id])
+
+        if editable_issues.empty?
+          comment_result = client.add_comment(ci_source.repo_slug, ci_source.pull_request_id, body)
+        else
+          original_id = editable_issues.first[:id]
+          comment_result = client.update_comment(ci_source.repo_slug, original_id ,body)
```

Comment:
Line is too long. [88/80]<br>Space missing after comma.<br>Space found before comma.